subscription-service/config/detekt/baseline.xml

```diff
@@ -7,7 +7,6 @@
     <ID>MaxLineLength:SubscriptionService.kt$SubscriptionService$ AND ( string_to_array(watched_attributes, ',') &amp;&amp; string_to_array(:updatedAttributes, ',') OR watched_attributes IS NULL )</ID>
     <ID>MaxLineLength:WebSecurityTestConfig.kt$WebSecurityTestConfig : WebSecurityConfig</ID>
     <ID>ReturnCount:QueryUtils.kt$QueryUtils$fun extractGeorelParams(georel: String): Triple&lt;String, String?, String?&gt;</ID>
-    <ID>SpreadOperator:WebConfig.kt$WebConfig$(*it.toTypedArray())</ID>
```

Comment:
no file has been modified in subscription service, Detekt baseline should not have been modified